FirebaseAdmin/FirebaseAdmin/Auth/FirebaseUserManager.cs

```diff
@@ -62,14 +62,19 @@ public static FirebaseUserManager Create(FirebaseApp app)
         }
 
         /// <summary>
-        /// Get a user by id.
+        /// Gets the user data corresponding to the given user ID.
         /// </summary>
-        /// <param name="uid">The query user id.</param>
+        /// <param name="uid">A user ID string.</param>
         /// <param name="cancellationToken">For asynchronous operation.</param>
         /// <returns>A record of user with the queried id if one exists.</returns>
         public async Task<UserRecord> GetUserById(
             string uid, CancellationToken cancellationToken = default(CancellationToken))
         {
+            if (string.IsNullOrEmpty(uid))
+            {
+                throw new ArgumentException("Failed to get user. User id cannot be null or empty");
```

Comment:
I think just "User ID cannot be null or empty." is sufficient. Also remember the fullstop at the end. Here and below.